tables/scripts/ptrepack.py

```diff
@@ -422,8 +422,15 @@ def main():
         )
 
     # Catch the files passed as the last arguments
-    src = args.src.split(':')
-    dst = args.dst.split(':')
+    if 'win' in sys.platform():
+        # the file separator for windows is deemed ','
+        src = args.src.split(',')
+        dst = args.dst.split(',')
+    else:
+        # the file separator for osx, linux for  is deemed ':'
+        src = args.src.split(':')
```

Comment:
I just realised `rsplit(':', 1')` was your original commit 476a2f956322569efefa5d779b457d639b90a552

Sorry about that, but I think it's the way to go...